skrf/calibration/deembedding.py

```diff
@@ -973,6 +973,14 @@ class IEEEP370_SE_NZC_2xThru(Deembedding):
         -|1  2|---|1  2|---|2  1|-
          +----+   +----+   +----+
          
+
+    Warning
+    -------
+    There are two differences compared to the original matlab implementation:

```

Comment:
is there a reference available for the Matlab implementation (a link?)

Reply:
yes, it is in the sources below the warning, but I will add it in the comment